LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -46,14 +70,49 @@ struct FHIRResourcesView: View {
                     }
                 }
                 .toolbar {
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button(action: {
+                            if !interpretingMultipleResources {
+                                showMultipleResourcesChat.toggle()
+                            } else {
+                            }
+                        }) {
+                            if interpretingMultipleResources {
+                                ProgressView("Interpreting...")
```

Comment:
Please use localized strings using the `Localizable.strings` file when you add new strings to the application.

Reply:
I removed this portion of the code because I added a "Chat with all records" button to the top of the list which uses a ```OnboardingActionsView``` section.